rpc_util.go

```diff
@@ -323,6 +324,25 @@ func (o MaxSendMsgSizeCallOption) before(c *callInfo) error {
 }
 func (o MaxSendMsgSizeCallOption) after(c *callInfo) {}
 
+// CallCompressionFactor returns a CallOption which sets the predicted compression factor,
+// used in sizing buffers
```

Comment:
Please add:

```go
//
// This API is EXPERIMENTAL.
```

We should call out where this applies (i.e. it's only when decompressing), and how it works (a reasonable interpretation of compression factor could be: "0.5" indicates that compression would halve the input).